codegen/smithy-aws-swift-codegen/src/test/kotlin/software/amazon/smithy/aws/swift/codegen/PresignerGeneratorTests.kt

```diff
@@ -73,9 +74,10 @@ class PresignerGeneratorTests {
             """
             import AWSClientRuntime
             import ClientRuntime
+            import Foundation
```

Comment:
would this work on Linux?

We should do partial import `import struct Foundation.TimeInterval`

Reply:
will verify and will do!